### ISSUE TITLE
Defining 24bit constants in included file does not work.

### ISSUE THREAD

**Issue by avalonbits** [envenomator/agon-ez80asm#1]
This program works as expected:

```asm
        .assume adl = 1
        .org $40000

        jr _start

_24bit_equ:         .equ  0FF00h
_exec_name:          .db  "EXEC.BIN",0

         align 64
        .db "MOS", 0,1

_start:
        ld hl, _24bit_equ
        ld hl, 0
        ret
```

But, if instead I put the constant in different file, I get an error

`equs.inc`
```asm
inc_24bit:        .equ 0FF00h
```

`prog.s`
```asm
        .assume adl = 1
        .org $40000
        .include "equs.inc"

        jr _start

_exec_name:          .db  "EXEC.BIN",0

         align 64
        .db "MOS", 0,1

_start:
        ld hl, inc_24bit
        ld hl, 0
        ret
```

When running the assumbler, I get the error:
```
"equs.inc"  - line 1 - Invalid mnemonic
```

If instead I define an 8 bit constant e.g. 85h, it works.

**Comment by envenomator** [comment]
I copied this exact code and it works fine, until I deliberately insert a space in before the 'inc_24bit:' label, then I get the exact error you are getting because then it isn't a label anymore and the error for invalid mnemonic is presented, as it should. Can you send me your exact files, as you have them on your SD card?

**Comment by avalonbits** [comment]
[bug01.zip](https://github.com/envenomator/agon-ez80asm/files/10901869/bug01.zip)

Here is the sources I'm using that triggers the error for me.

**Comment by avalonbits** [comment]
This was fixed by commit e73be3c
